plugins/default/lib/openshift.py

```diff
@@ -5,18 +5,35 @@
 def oc(*args):
     return check_output(("oc",) + args)
 
-def _get_running_pod_names(oc, project):
+
+def _get_running_pods(oc, project):
```

Comment:
I changed it because the original version you had the exported function name was calling the internal function name without the _names in it, I should have fixed the internal name instead of changing the call.  This is now fixed.